clients/harbor-client/deploy/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.configSecretName }}
+  labels:
+    {{- include "snappcloud-harbor-client.labels" . | nindent 4 }}
+type: Opaque
+data:
```

Comment:
Using `stringData` could be better, because base64 encoding is handled by the k8s client automatically.